modules/cert-manager-istio-creating-issuer.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-istio-creating-issuer_{context}"]
+= Creating the issuer for the {cert-manager-operator} with Istio-CSR
+
+Use this procedure to create the issuer for
+
+[NOTE]
+====
+Other supported issuers can be used, except for the ACME issuer, which is not supported. For more information, see "cert-manager issuer types".
```

Comment:
I would add an additional resources section in the assembly, under this module, to link to the section you're referring to here

Reply:
+1 (same as https://github.com/openshift/openshift-docs/pull/89608#discussion_r1993980069